cmd/govim/testdata/scenario_default/quickfix_fallback_directory.txt

```diff
@@ -0,0 +1,143 @@
+# Test that the quickfix window logic fallbacks to first error of the selected
+# entry directory, if this one is not found in the new quickfix list and if
+# there's no other errors in this entry file.
+
+vim ex 'e charly.go'
+vimexprwait errors.golden.orig GOVIMTest_getqflist()
+
+# Verify we have the first entry selected
+# NOTE: disabled because when errors are spreaded accross multiple files like 
```

Comment:
I would delete the commented out test, because as you say, we can't assert anything due to the random order of diagnostics arriving. But then I would keep a comment explaining how such a situation can come about. 

FYI: s/spreaded/spread/ in your sentence.

Reply:
ok removed e33cf50